source/operations/monitoring/metrics.md

```diff
@@ -0,0 +1,137 @@
+# List of metrics reported cluster wide
```

Comment:
@djwfyi we usually don't check in imported files since they can change on build and get into other commits.

You can add this to `.gitignore` and then pop it out using `git rm --cached source/operations/monitoring/metrics.md` I believe.

Reply:
Arg. I meant to take care of that.  Thanks for the catch.